src/controller/handler/issue.credential.ts

```diff
@@ -85,7 +103,30 @@ export class IssueCredential implements IAgentResponseHandler {
             return res.data;
         }
 
-        Logger.info(`doing nothing for ${agentId}: route ${route}: topic ${topic}`);
+        if (body.role === 'issuer' && body.state === 'offer_sent') {
+            const action = 'issue';
+            const templatedCacheKey = `${agentId}-${body.role}-${body.credential_exchange_id}`;
+            await this.checkPolicyForAction(action, templatedCacheKey);
+            await readPermission(action, templatedCacheKey);
+
+            const url: string = agentUrl + `/${IssueCredential.ISSUE_CREDENTIALS_URL}/records/${body.credential_exchange_id}/${action}`;
+            const data = {
+                'comment': 'no comment'
```

Comment:
Optional: if we wanted to make use of this field we could put something like 'guardianship' or 'Agent in guardianship', not super useful but it's something